### ISSUE TITLE
Order is not saved for editors after 3.4.1 security fix

### ISSUE THREAD

**Issue by theodejager** [hijiriworld/intuitive-custom-post-order#55]
@hijiriworld @timohubois 

After the last update, when editors re-order items, the new order is not saved.

The nonce verification that was added doesn't seem to pass (in the `update_menu_order*` functions)

```
		if ( ! wp_verify_nonce( $_POST['nonce'], 'hicpojs-ajax-nonce' ) ) {
			return;
		}
```

**Comment by timohubois** [comment]
@theodejager have you also tested it after clearing the browser cache?

**Comment by theodejager** [comment]
@timohubois good point, thanks, it seems to work on a clean browser.
I suspect I'm not the only one with the problem so I made a pull request to better enqueue the JS/CSS files to prevent caching when they are modified.

https://github.com/hijiriworld/intuitive-custom-post-order/pull/56

**Comment by timohubois** [comment]
@theodejager great to hear that it works and thanks for the pr. Preventing this issue is currently also included in #54.
